src/distances/strings.rs

```diff
@@ -1,4 +1,5 @@
 use crate::core::number::Number;
+use crate::distances::alignment_helpers;
```

Comment:
`use super::alignment_helpers;`

Reply:
What is the rationale behind this? I've seen things done in both ways throughout CLAM and never understood why.